RELEASING.md

```diff
@@ -46,30 +44,31 @@ git compare EGI-Foundation 0.8.3..master
 git log --abbrev-commit 0.8.3..master
 # Prepare a PR to prepare version (0.9.0 here)
 git checkout -b prepare-0.9.0
-# Prepare a complete changelog and add it to the issue as reference
+# Prepare the changelog and add it as a new entry to CHANGELOG
+vim CHANGELOG
 # Take care to changes in dependencies and configuration
 # Depending on the changes update relevant packages
-# Debian: debian/changelog: add an entry at the top
 # Debian: dependencies in debian/control files
 # Debian: debs/cloud-info-provider-opennebula/debian/changelog
 # Debian: debs/cloud-info-provider-openstack/debian/changelog
-# RHEL: bump version and dependencies at the top, add entry at the bottom
+# RHEL: change dependencies at the top
 # RHEL: rpm/cloud-info-provider.spec
 # RHEL: rpm/cloud-info-provider-opennebula.spec
 # RHEL: rpm/cloud-info-provider-openstack.spec
+# Update Zenodo configuration if needed
 # Update AUTHORS file if needed
 vim AUTHORS
-# Update Zenodo configuration
-# Description, title, version, publication_date, creators
-vim .zenodo.json
 # Commit changes
 git commit -am 'Prepare release 0.9.0'
-# Push new branch to fork
-git push --set-upstream origin prepare-0.9.0
 # Create pull request
-git pull-request
+gh pr create
 # After PR merging
-git release create -m 0.9.0 0.9.0
+curl \
```

Comment:
Maybe would be nice to have a comment/reference somewhere to where to create/get the token (pointing to official doc or githug settings URL maybe).

Reply:
👍